### PR TITLE
13907 cannot complete mhr registration bug

### PR DESCRIPTION
**Pull request by ChasBoucher**
*Issue #:* /bcgov/entity#13907

*Description of changes:*

- [ ] Handles api errors and ensure the resetallvaidations does not get executed
- [ ] Add "Unable to complete registration" dialog box if error occurs.

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the PPR license (Apache 2.0).

### CODE REVIEW

File: ppr-ui/src/views/newMhrRegistration/MhrRegistration.vue [hunk 1]
```diff
@@ -161,6 +171,10 @@ export default defineComponent({
       })
     }
 
+    const handleDialogResp = () => {
+      localState.showCancelDialog = false
+    }
```
Comment (cameron-eyds) [inline ppr-ui/src/views/newMhrRegistration/MhrRegistration.vue]: Simple enough that you could do this in the template on the event. 

ie line 9 ` @proceed="showCancelDialog = false"`

**Review by cameron-eyds — APPROVED** [review]
Looks good to me, just a small comment.

File: ppr-ui/src/views/newMhrRegistration/MhrRegistration.vue (hunk 1 above)
Comment (ChasBoucher) [reply ppr-ui/src/views/newMhrRegistration/MhrRegistration.vue]: Makes sense..  change made

**Review by doug-lovett — APPROVED** [review]
LGTM

**Review by dimak1 — APPROVED** [review]

**Pull request merged**